### PR TITLE
Setup-desktop version

### PR DESCRIPTION
**Pull request by dhirajsingha143**
- Update the README file with recent changes.
- Increase compatibility of portfolio site on bigger screens.

### CODE REVIEW

**Comment by dhirajsingha143** [comment]
- Linter errors resolve.

File: index.html [hunk 1]
```diff
@@ -9,16 +9,24 @@
...
     <header class="header">
       <nav class="nav container-padding-x container">
         <a class="logo" href="#" target="_blank">My Logo</a>
+        <div class="menu">
+          <a class="link" href="#portfolio" target="_self">Portfolio</a>
+          <a class="link" href="#about" target="_self">About</a>
+          <a class="link" href="#contact" target="_self">Contact</a>
+        </div>
         <i class="fa-solid fa-bars"></i>
       </nav>
```
Comment (iambenkis) [inline index.html:22]: - [ ] Make sure the space between the `logo` and the `nav-list` should stick to the Figma design, It's an important part of the requirement. Kindly fix that. :+1: 

Your design

Screenshot from 2022-12-08 23-14-30

Figma Design

Screenshot from 2022-12-08 23-16-10

File: index.html [hunk 2]
```diff
@@ -160,25 +182,27 @@ <h2 class="title">About<br />Myself</h2>
...
+          </div>
         </div>
       </section>
     </main>
-    <footer class="contact">
+    <!-- Fotter Section -->
+    <footer id="contact" class="contact">
       <div class="container container-padding-x">
         <h2 class="contact-title">Contact me</h2>
         <p class="contact-text">
```
Comment (iambenkis) [inline index.html:208]: - [ ] Good job for this job you did :+1: , however , make sure the contact `section` fill all the width of the screen in the `desktop` version as it is represented in the Figma design. Kindly remove the space in `Red`. :100: on the picture below

Your design 

Screenshot from 2022-12-08 22-59-31

File: index.html [hunk 3]
```diff
@@ -51,89 +59,103 @@ <h1 class="title">I'm Dhiraj<br />Glad to see<br />you!</h1>
...
+      <!-- About Section -->
+      <section id="about" class="about container-padding-x container">
+        <div class="social-container">
           <div class="about-social-container">
+            <h2 class="title">About<br />Myself</h2>
+            <p class="text">
+              Hello I’m a software developer! I can help you build a product , feature or website Look through some of my
+              work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact
+              me.
+            </p>
```
Comment (iambenkis) [inline index.html:158]: - [ ] Good job for this job you did :+1: , however , make sure the  ` about section` fill all the width of the screen in the `desktop` version as it is represented in the Figma design. Kindly remove the space in `Red`. :100: on the picture below

Your design 

Screenshot from 2022-12-08 23-11-43

**Review by iambenkis — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ 

## Hi @dhirajsingha143  👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project, but you are almost there!

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- Gitflow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark: 



 ## Required Changes ♻️

- [ ] You should stick to the design as much as possible (e.g. font, colours, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1). Kindly make sure you respect the width of the sections :+1: . _**You will learn more as you look at the comments below.**_

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me `@iambenkis` in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by dhirajsingha143** [comment]
- Resolve Navigation spacing.
- Resolve spacing issues (Now compatible with screens up to 4000px width)

**Review by Tadesse-Alemayehu — APPROVED** [review]
### Hi @dhirajsingha143 

### I'm Tadesse Alemayehu your reviewer for this PR. :hand: 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
68747470733a2f2f6669636f666f72756d732e6d796669636f2e636f6d2f74352f696d6167652f736572766572706167652f696d6167652d69642f343932363069463839303835363833444139334135422f696d6167652d73697

## Highlights
 - [ ] Good job making the changes requested by the previous code reviewer. 🙏
 - [ ]  Correct GitHub Flow 👍🏾
 - [ ]  Good commit history 🥇
 - [ ]  Good PR title/description 💯
 - [ ]  All linters passed 🥇
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**